### ISSUE TITLE
[BUG] Schematic Load java.lang.NullPointerException

### ISSUE THREAD

**Issue by marvinej** [boy0001/FastAsyncWorldedit#1198]
# Bug report

**Debug paste link**:
`[Sun, 11. Nov 2018 17:50:35 EST ERROR] [WorldEdit] An unexpected error occurred while handling a FAWE command
[Sun, 11. Nov 2018 17:50:56 EST INFO] SirKratosAurion issued server command: /worldedit debugpaste
[Sun, 11. Nov 2018 17:51:03 EST INFO] (FAWE) There was an error handling a FAWE command: [See console]
[Sun, 11. Nov 2018 17:51:03 EST INFO] java.io.IOException: Server returned HTTP response code: 503 for URL: https://hastebin.com/documents
[Sun, 11. Nov 2018 17:51:03 EST ERROR] [WorldEdit] An unexpected error occurred while handling a FAWE command`

**Description of the problem:**
https://pastebin.com/Y04ZP0L2

**How to replicate**:

I copped and //schem saved the maze from one server, moved it to the other and tried to //schem load it into the other server. same version of FAWE 

**Checklist**:

- [NOT WORKING] I included a `/fawe debugpaste` link
- [X] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/boy0001/FastAsyncWorldedit/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of FAWE
FastAsyncWorldEdit-bukkit-18.10.26-94d5028-2069
- [x] I Made sure the bug/error is not caused by any other plugin

**Comment by IMegaLightI** [comment]
[21:29:27 WARN]: java.lang.NullPointerException
[21:29:27 WARN]: at com.sk89q.minecraft.util.commands.CommandContext.split(CommandContext.java:43)
[21:29:27 WARN]: at com.sk89q.worldedit.util.command.parametric.StringArgumentStack.<init>(StringArgumentStack.java:62)
[21:29:27 WARN]: at com.sk89q.worldedit.util.command.parametric.AParametricCallable.getScopedContext(AParametricCallable.java:50)
[21:29:27 WARN]: at com.sk89q.worldedit.util.command.parametric.AParametricCallable.getSuggestions(AParametricCallable.java:234)
[21:29:27 WARN]: at com.sk89q.worldedit.util.command.SimpleDispatcher.getSuggestions(SimpleDispatcher.java:174)
[21:29:27 WARN]: at com.sk89q.worldedit.util.command.SimpleDispatcher.getSuggestions(SimpleDispatcher.java:174)
[21:29:27 WARN]: at com.sk89q.worldedit.extension.platform.CommandManager.handleCommandSuggestion(CommandManager.java:551)
[21:29:27 WARN]: at sun.reflect.GeneratedMethodAccessor394.invoke(Unknown Source)
[21:29:27 WARN]: at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:29:27 WARN]: at java.lang.reflect.Method.invoke(Method.java:498)
[21:29:27 WARN]: at com.sk89q.worldedit.util.eventbus.MethodEventHandler.dispatch(MethodEventHandler.java:58)
[21:29:27 WARN]: at com.sk89q.worldedit.util.eventbus.EventHandler.handleEvent(EventHandler.java:73)
[21:29:27 WARN]: at com.sk89q.worldedit.util.eventbus.EventBus.dispatch(EventBus.java:187)
[21:29:27 WARN]: at com.sk89q.worldedit.util.eventbus.EventBus.post(EventBus.java:173)
[21:29:27 WARN]: at com.boydti.fawe.bukkit.listener.ATabCompleteListener.onTab(ATabCompleteListener.java:33)
[21:29:27 WARN]: at com.boydti.fawe.bukkit.listener.SyncTabCompleteListener.onTabComplete(SyncTabCompleteListener.java:19)
[21:29:27 WARN]: at sun.reflect.GeneratedMethodAccessor393.invoke(Unknown Source)
[21:29:27 WARN]: at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[21:29:27 WARN]: at java.lang.reflect.Method.invoke(Method.java:498)
[21:29:27 WARN]: at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304)
[21:29:27 WARN]: at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[21:29:27 WARN]: at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500)
[21:29:27 WARN]: at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485)
[21:29:27 WARN]: at org.bukkit.craftbukkit.v1_13_R2.CraftServer.tabComplete(CraftServer.java:1668)
[21:29:27 WARN]: at org.bukkit.craftbukkit.v1_13_R2.command.BukkitCommandWrapper.getSuggestions(BukkitCommandWrapper.java:49)
[21:29:27 WARN]: at com.mojang.brigadier.tree.ArgumentCommandNode.listSuggestions(ArgumentCommandNode.java:68)
[21:29:27 WARN]: at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:379)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:516)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.PacketPlayInTabComplete.a(SourceFile:36)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.PacketPlayInTabComplete.a(SourceFile:9)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10)
[21:29:27 WARN]: at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[21:29:27 WARN]: at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835)
[21:29:27 WARN]: at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733)
[21:29:27 WARN]: at java.lang.Thread.run(Thread.java:748)

**Comment by Combustible** [comment]
We've been really hurting for this bug too. It seems to only happen when any of the things in your selection contain NBT data (chests, signs, command blocks, spawners, etc.). If you only have blocks, it works fine. 

You might be able to use the "structure" format until this gets fixed (though it lags like hell and the files are huge): 
```
//copy
//schematic save structure temp
...
//schematic load structure temp
//paste
```

The upper limit of this for me seems to be about 2 million blocks before it just won't save anymore (server crashes). Is extremely RAM dependent - more RAM seems to mean more saving before crash. Loading has the same problem. 

The biggest problem with the structure format is that blockstates aren't saved correctly so stairs/etc are screwed up / facing the wrong way. :(

**Comment by robotnikthingy** [comment]
Having the same issue as well. This is the schematic I am trying to paste

https://we.tl/t-gQzmwFWUA4

**Comment by N0tMyFaultOG** [comment]
Your ticket has been closed because it falls into one of the following categories:

- Your issue has been fixed in a newer version of FastAsyncWorldEdit for versions 1.8 - 1.12.2, get the latest build [here](https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/target/) matching the platform you need.

- Your issue targets minecraft version 1.13 or higher. This is the wrong repository then. Move to [this repository](https://github.com/IntellectualSites/FastAsyncWorldEdit) and open a bug report here.
BUT BEWARE: You should consider updating to 1.15.2, using the latest Fawe build for 1.15.2 (Build 255 when writing this, get it [here](https://ci.athion.net/job/FastAsyncWorldEdit-1.15/)) and reproducing the issue in a clean environment (fresh setup Spigot or Paper server).

- Your issue has been fixed in a build of FAWE for 1.15 or an issue exists on our new repository.

- You either didn't fill out the template or deleted it.

If none of the bullet points suits, head to the second one.
